tests/install_testbed/create_mysqldb

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -x
+# Copyright 2015 Telefónica Investigación y Desarrollo, S.A.U
+#
+# This file is part of FIWARE project.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+#
+# You may obtain a copy of the License at:
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For those usages not covered by the Apache version 2.0 License please
+# contact with opensource@tid.es
+#
+# Author: Chema
+
+source ./config_vars
+
+sudo mysql -u root -p$MYSQL_PASS <<EOF
+CREATE DATABASE $1;
+GRANT ALL PRIVILEGES ON $1.* TO '$1'@'localhost' IDENTIFIED BY '$2';
+GRANT ALL PRIVILEGES ON $1.* TO '$1'@'%' IDENTIFIED BY '$22';
```

Comment:
You're right, the line is wrong. The problem was not detected because all the connections are from localhost, but with other IP this configuration does not work.